src/routes/assemble.js

```diff
@@ -61,8 +61,12 @@ if (config) {
 }
 
 const checkPresenceOf = function (req, res, next) {
-  const { guideId, fileDataUrl } = req.body
+  const { answers, fileDataUrl, guideId } = req.body
 
+  if (!answers) {
+    return res.status(400)
+      .send('You must provide answers')
```

Comment:
should this say something like `at least an empty set of answers`? This message is really for us, so it probably doesn't matter that much.

Reply:
won't this work as long as there is an answers key even if it is empty. This seems correct if it will fail on no anwsers key only

Reply:
I expanded the validation logic so there’s even better error messages returned.